Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -48,6 +53,8 @@ final class ExperimentalAnimationLayer: CALayer {
     var timeOffset: TimeInterval = 0
   }
 
+  var imageProvider: AnimationImageProvider
```

Comment:
This needs to be a `var` because it implements `var imageProvider: AnimationImageProvider { get set }` from `RootAnimationLayer`.

I added a `didSet` to make sure any existing `ImageLayer`s are reconfigured using the new `AnmationImageProvider` (and tweaked the tests so it fails without this).